charts/images.yaml

```diff
@@ -2,7 +2,7 @@ images:
 - name: etcd-backup-restore
   sourceRepository: github.com/gardener/etcd-backup-restore
   repository: eu.gcr.io/gardener-project/gardener/etcdbrctl
-  tag: "v0.13.0"
+  tag: "TODO"
```

Comment:
Is this `TODO` supposed to be here?

Reply:
Yes, this is just a reminder to switch to the proper version once we release the copy operation in the etcd backup restore.